examples/todolist-react/src/hooks/useInput.js

```diff
@@ -3,9 +3,9 @@ import { useCallback, useState } from "react";
 export default function useInput(defaultValue) {
   const [value, setValue] = useState(defaultValue || "");
 
-  const onChange = useCallback(event => {
+  const onChange = useCallback((event) => {
```

Comment:
In order not to recreate the function all the time, we just pass it in an empty array